### PR TITLE
MM-18903 Removal of redux requests states for integrations object

### PR DESCRIPTION
**Pull request by nadalfederer**
#### Summary

Remove redux requests which are not needed

#### Ticket Link
Fixes https://github.com/mattermost/mattermost-server/issues/12519

Awaiting PR on https://github.com/mattermost/mattermost-redux/pull/1006

### CODE REVIEW

**Comment by nadalfederer** [comment]
Is this the right approach? Im not sure

File: components/integrations/add_oauth_app/index.js [hunk 1]
```diff
@@ -7,9 +7,8 @@ import {addOAuthApp} from 'mattermost-redux/actions/integrations';
 
 import AddOAuthApp from './add_oauth_app.jsx';
 
-function mapStateToProps(state) {
```
Comment (sudheerDev) [inline components/integrations/add_oauth_app/index.js]: We can remove the `mapStateToProps `as we dont need it anymore and pass in null to `connect ` below.

Comment (nadalfederer) [reply components/integrations/add_oauth_app/index.js]: fixed it

**Comment by jasonblais** [comment]
@sudheerDev / @jespino looks like this PR is now ready to merge as the related PR (https://github.com/mattermost/mattermost-redux/pull/1006) is merged

**Pull request merged**